rest-service/manager_rest/celery_client.py

```diff
@@ -27,11 +28,27 @@
 class CeleryClient(object):
 
     def __init__(self):
-        amqp_uri = 'amqp://{0}'.format(config.instance().amqp_address)
+        # Use the encrypted connection if supplied
+        if config.instance().amqp_ca_path != '':
+            ssl_settings = {
+                'ca_certs': config.instance().amqp_ca_path,
+                'cert_reqs':  ssl.CERT_REQUIRED,
+            }
+        else:
+            ssl_settings = None
+
+        amqp_uri = 'amqp://{username}:{password}@{address}'.format(
```

Comment:
Annoyingly, the code that delivers the 'address' to here delivers it complete with port. I didn't have time to dig further into it, but it probably wants a comment to that effect.